RELEASE.md

```diff
@@ -43,14 +47,15 @@ It should look like:
     $ git commit -a -m "Bump version to $VERSION"
     $ git neco review
     ```
+
 7. Make sure that Sonobuoy test has been passed when updating to `x.y.0` and its RC.
 8. Merge the pull request.
-9. Make a tag and push it.
+9. Add a git tag to the main HEAD, then push it.
 
     ```console
     $ git checkout main
     $ git pull
-    $ git tag "v$VERSION"
+    $ git tag -a "v$VERSION"
```

Comment:
@ymmt2005 
What message should we write for the tag?
`git tag -a "v$VERSION" -m ""` may be better if nothing is needed.

Reply:
I'd leave it to each developer.
Personally, I'd write like this:

`Release $VERSION`

Reply:
It should be confusing for not explaining what kind of info to write.
If no meaningful message is needed, I personally like to write the instruction as `git tag -a "v$VERSION" -m "Release $VERSION"`.
Otherwise 
```
$ git tag -a "v$VERSION"
(Write any relevant information)
$ git push origin "v$VERSION"
```
looks good.

Reply:
will do